minecraft_servers/kanmuru/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "server_name": "kanmuru",
+  "nice_name": "Kanmuru",
+  "direct_ip": "kanmuru.net",
+  "server_wildcards": [
+    "kanmuru.net",
+    "%.kanmuru.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://kanmuru.net",
+    "discord": "https://discord.gg/FVF95pXbYy",
+    "twitter": "KanmuruNET"
+  },
+  "discord": {
+    "server_id": 826000371407257616,
+    "rename_to_minecraft_name": true
```

Comment:
Please use `false`, as this is the default value and enabling is only allowed for partners.